docs/query-language.md

```diff
@@ -420,3 +420,83 @@ Range(frame="stargazer", stargazer_id=1, start="2010-01-01T00:00", end="2017-03-
 Returns `{{"attrs":{},"bits":[10]}`
 
 * bits are repositories which were starred by user 1 from 2010-01-01 to 2017-03-02
+
+
+#### Range (BSI)
+
+**Spec:**
+
+```
+Range(<frame=STRING>, <FIELD_NAME, COMPARISON_OPERATOR, integer> )
+```
+
+**Description:**
+
+The to `Range` query is overloaded to also work on `field` values.
+Returns bits that are true for the comparison operator
+
+**Result Type:** object with attrs and bits
+
+
+**Examples:**
+
+In our source data commitactivity was counted over the last year.
+This Range query finds all repositories that had that many commits.
+
+```
+Range(frame="stats", commitactivity > 100)
+```
+
+Returns `{{"attrs":{},"bits":[10]}`
+
+* bits are repositories which had at least 100 commits in the last year.
+
+
+#### Sum
+
+**Spec:**
+
+```
+Sum(<frame=STRING>, <field=STRING>)
+```
+
+**Description:**
+
+Returns the computed sum of all `field` values across the bits in a `frame` plus the count of bitmaps with a field value.
+
+**Result Type:** object with sum and count.
```

Comment:
these result type descriptions are pretty vague. can we be a little more specific.